core/processor.go

```diff
@@ -86,14 +88,20 @@ func (p *Processor) Start() {
 			// Solana
 			watcher = solana.NewWatcher(cfg, p.db, p.txsCh, p.txTrackCh)
 			dispatcher = solana.NewDispatcher(cfg.Rpcs, cfg.Wss)
+		} else if chain == "lisk-testnet" {
```

Comment:
get the latest version on lib repo. Run 

go get github.com/sisu-network/lib@COMMIT_HASH
go mod tidy 

After that, you can replace this line with `libchain.IsLiskChain`

Reply:
Thank a